src/api/custom-resolvers/mutations/update-inactive-applications/index.ts

```diff
@@ -0,0 +1,43 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import { SuccessResponse } from '../../../types';
+import getInactiveApplications from '../../../helpers/get-inactive-applications';
+import mapAndUpdateInactiveApplications from '../../../helpers/map-and-update-inactive-applications';
+
+/**
+ * getInactiveApplications
+ * Gets inactive applications which have not been updated for 30 days
+ * Sets their status to Abandoned
+ * returns success flag
+ * @param {Object} GraphQL root variables
+ * @param {Object} KeystoneJS context API
+ * @returns {Promise<SuccessResponse>} success flag
+ */
+const updateInactiveApplicationsMutation = async (root: any, context: Context): Promise<SuccessResponse> => {
+  try {
+    console.info('Getting and updating inactive applications');
+
+    /**
+     * gets inactive applications based on status and updatedAt
+     * returns array of application ids and statuses
+     */
+    const applications = await getInactiveApplications(context);
```

Comment:
Maybe should change the comment to simply "get inactive applications"

Just thinking, if `getInactiveApplications` changes to something different, this comment could become misleading